### DISCUSSION TITLE
Import from WordPress

### DISCUSSION THREAD

**Discussion by bibicadotnet** [ArtalkJS/Artalk#638] [Q&A]
How to import data from WordPress

Instructions from [artransfer ](https://artransfer.netlify.app/ )are currently inactive

**Comment by qwqcode** [comment]
We have identified an issue occurring when a WordPress page contains only one comment. This problem has been rectified. Thank you very much for bringing it to our attention!

Refer to: https://github.com/ArtalkJS/Artransfer/pull/5

**Reply by bibicadotnet** [reply]
Yes, before that I temporarily fixed it by moving WordPress -> Commento -> Artalk
